gql_code_builder/lib/src/operation/data.dart

```diff
@@ -32,9 +32,25 @@ List<Class> _buildOperationDataClasses(
       doc,
       schema,
       schemaUrl,
-      "Query",
+      _operationType(
+        schema,
+        op,
+      ),
     );
 
+String _operationType(
+  DocumentNode schema,
+  OperationDefinitionNode op,
+) {
+  final schemaDefs = schema.definitions.whereType<SchemaDefinitionNode>();
```

Comment:
I wish Dart analyzer had a rule for empty line before return.

```suggestion:-0+7
  final schemaDefs = schema.definitions.whereType<SchemaDefinitionNode>();
  
  if (schemaDefs.isEmpty) return defaultRootTypes[op.type];
  
  return schemaDefs.first.operationTypes
      .firstWhere(
        (opType) => opType.operation == op.type,
      ).type.name.value;
```

Reply:
done